blocks/hero/hero.css

```diff
@@ -1,40 +1,213 @@
-/* block specific CSS goes here */
-
-main .hero-container > div {
-    max-width: unset;
-}
-  
-main .hero-container {
-    padding: 0;
-}
-  
-main .hero {
-    position: relative;
-    padding: 32px;
-    min-height: 300px;
+.hero-container {
+    padding: 0 !important;
+    font-family: var(--hero-text-font-family);
+    max-width: 540px;
+    margin: 0 auto;
 }
 
-main .hero h1 {
-    max-width: 1200px;
-    margin-left: auto;
-    margin-right: auto;
-    color: white;
+.hero-wrapper {
+    max-width: max-content;
+    width: 100%;
+    box-sizing: border-box;
 }
 
+.hero {
+    padding: 0 15px;
+}
 
-main .hero picture {
+.hero picture {
+    background-size: cover;
     position: absolute;
-    z-index: -1;
-    top: 0;
+    z-index: -2;
+    top: 70px;
     left: 0;
     bottom: 0;
     right: 0;
     object-fit: cover;
     box-sizing: border-box;
 }
 
-main .hero img {
+.hero img {
     object-fit: cover;
     width: 100%;
-    height: 100%;
+    height: 353px;
+}
+
+.hero h1 {
+   font-family: var(--heading-font-family);
+   text-transform: uppercase;
+   font-size: var(--heading-font-size-xl);
+   line-height: 50px;
+   font-weight: 600;
+   margin-bottom: 9px;
+   margin-top: 0;
+   padding-top: 150px;
+}
+
+.hero h1 + h3 {
+    font-family: var(--hero-text-font-family);
+    font-size: var(--heading-font-size-xs);
+    line-height: 25px;
+    text-transform: uppercase;
+    font-weight: 500;
+    margin-bottom: 18px;
+    letter-spacing: 0.4px;
+    margin-top: 0;
+    padding-right: 15px;
+}
+
+.hero div:first-child div {
+    height: 353px;
```

Comment:
This line causes problems on Safari (iPhone SE). Change to `auto` fix the issue
<img width="317" alt="obraz" src="https://user-images.githubusercontent.com/125962117/221868444-05ed410f-c2f1-45b8-8ede-05977820fd74.png">